package.json

```diff
@@ -27,10 +27,10 @@
   },
   "homepage": "https://github.com/dashevo/DashJS#readme",
   "dependencies": {
-    "@dashevo/dapi-client": "~0.11.0",
+    "@dashevo/dapi-client": "~0.12.0",
     "@dashevo/dashcore-lib": "^0.18.0",
-    "@dashevo/dpp": "~0.11.1",
     "@dashevo/wallet-lib": "^6.0.1",
```

Comment:
Used for testing: `"@dashevo/wallet-lib": "git://github.com/dashevo/wallet-lib.git#develop",`